Makefile

```diff
@@ -138,7 +139,7 @@ test: manifests generate fmt vet envtest ## Run tests.
 
 .PHONY: e2etest
 e2etest: manifests generate local-deploy chainsaw
-	$(CHAINSAW) test ./e2e
+	echo "name: capl-e2e-$(GIT_REF)" | $(CHAINSAW) test ./e2e --values -
```

Comment:
could this be supplied as just a flag rather than piping it to the command?

Reply:
No, we only have the following options: https://kyverno.github.io/chainsaw/latest/configuration/values/#example

Reply:
I asked the chainsaw devs and they mentioned there is an `env()` function we could use to just read env variables in the tests, perhaps we could do that instead to just read in the GIT_REF?

Reply:
Unfortunately, Chainsaw seems to run tests in something like a subshell (e.g. `$USER` is accessible) so Make variables are not available.